client/public/locales/fr.json

```diff
@@ -208,7 +215,17 @@
           "tooltip": "Sélectionne un objet, puis contrôle les actions possibles sur celui-ci. Attention : la couleur et la température d'une lampe ne sont pas compatibles.",
           "color": "Couleur",
           "brightness": "Intensité",
-          "temperature": "Température"
+          "temperature": "Température",
+          "lightTurnOn": "lampe : turn_on",
+          "lightTurnOff": "lampe : turn_off",
+          "lightToggle": "lampe : toggle",
+          "switchTurnOn": "prise : turn_on",
+          "switchTurnOff": "prise : turn_off",
+          "switchToggle": "prise : toggle"
```

Comment:
Is it possible to write it in french ?
-- > "lampe : allumer"

Reply:
no, I wanted the verb to be coherent with the other actions